README.md

```diff
@@ -1,67 +1,91 @@
 Set up Dwelling Flask Testing Backend (for the first time)
-NOTE: no database setup here. Backend uses list-based live data 
+NOTE: Database is SQLite3 via SQLAlchemy 
+
++ Github Repo: {https://github.com/codeforpdx/dwellinglybackend}
++ Live Server: {https://dwellinglyapi.herokuapp.com/ } 
+
+### To Start Server
 
 1. Fork the backend and clone a copy ( https://help.github.com/en/github/getting-started-with-github/fork-a-repo )
```

Comment:
Is there a formal decision to start cloning the repository directly? As opposed to forking it first?